src/components/presentational/TimeSeriesChart/TimeSeriesChart.tsx

```diff
@@ -226,93 +171,117 @@ export default function TimeSeriesChart(props: TimeSeriesChartProps) {
         dataToDisplay = props.data;
     }
 
+    let yAxisDomain: AxisDomain | undefined = undefined;
+    if (props.options && props.chartType === "Line") {
+        let domainMin = (props.options as MultiSeriesLineChartOptions)?.domainMin;
+        if (domainMin === "Auto") {
+            yAxisDomain = ["auto", "auto"];
+        } else if (domainMin !== undefined) {
+            yAxisDomain = [domainMin, "auto"];
+        }
+    }
+
+    const xAxisTicks = getXAxisTicks();
+
     return <div className="mdhui-time-series-chart" ref={props.innerRef}>
         {props.title &&
             <CardTitle title={props.title}></CardTitle>
         }
         <div className="chart-container">
-            {(!props.chartHasData) &&
-                <div>
-                    {!!dataToDisplay &&
-                        <div className="no-data-label">No Data</div>
+            {!props.chartHasData && !!dataToDisplay && <div className="no-data-label">No Data</div>}
+            {!props.chartHasData && !dataToDisplay && <LoadingIndicator />}
+            <ResponsiveContainer width="100%" height={150}>
+                <ComposedChart data={dataToDisplay} syncId={props.syncId}>
+                    {props.chartHasData && props.tooltip &&
```

Comment:
The `tooltip` property is now optional.  If omitted, we will not add the `Tooltip` component to the chart.